alerts/config.go

```diff
@@ -0,0 +1,158 @@
+package alerts
+
+// Data models for care team alerts.
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/tidepool-org/platform/data/blood/glucose"
+	"github.com/tidepool-org/platform/structure"
+)
+
+// Config models a user's desired alerts.
+type Config struct {
+	OwnerID         string         `json:"ownerID" bson:"ownerID"`
+	InvitorID       string         `json:"invitorID" bson:"invitorID"`
```

Comment:
This should be the userId of the user that's being "followed". If the account is custodial (soon we'll introduce the ability for regular users to create custodial accounts) the invitor id and followee id might be different. I think it's better to revise the attribute name.

Additionally, I don't think we should use uppercase ID for consistency with other json/bson models.

Reply:
Re invitor vs follower, that makes sense, thanks for the clarification. 

What would you recommend for a the attribute name? `FolloweeId` feels stranged... `FollowedId` maybe? I'll go with that I think. It's a least a little less impersonal than `SourceId`, heh.